.circleci/config.yml

```diff
@@ -43,6 +43,13 @@ workflows:
           filters:
             branches:
               only: /^(?!pull\/).*$/
+      - cross_build:
```

Comment:
A new job to flush out problems immediately after merging, instead of waiting until midnight.